src/Repository/Doctrine/UserRepository.php

```diff
@@ -31,4 +31,10 @@ public function signUp(User $user): void
         $this->getDocumentManager()->persist($user);
         $this->getDocumentManager()->flush();
     }
+
+    public function save(User $user): void
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Update this method so that its implementation is not identical to "signUp" on line 29. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS4144)